<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="album_label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Album</string>
    </property>
    <widget class="QMenu" name="menuload_Folder">
     <property name="title">
      <string>load Folder</string>
     </property>
     <addaction name="actionsingle_Folder"/>
     <addaction name="actionrekursive_Folders"/>
    </widget>
    <addaction name="menuload_Folder"/>
    <addaction name="actionload_Picture"/>
    <addaction name="separator"/>
    <addaction name="actionclear_Album"/>
   </widget>
   <widget class="QMenu" name="pictureMenu">
    <property name="title">
     <string>Pictures</string>
    </property>
    <addaction name="showDefaultExifAct"/>
    <addaction name="clearDefaultExifAct"/>
    <addaction name="separator"/>
    <addaction name="showDefaultIptcAct"/>
    <addaction name="clearDefaultIptcAct"/>
    <addaction name="separator"/>
    <addaction name="selectAllImagesAct"/>
    <addaction name="writeDefaultMetaToSelectedImagesAct"/>
    <addaction name="separator"/>
    <addaction name="removeImagesAct"/>
   </widget>
   <widget class="QMenu" name="infoMenu">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="hw_infoAct"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="pictureMenu"/>
   <addaction name="infoMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionload_Picture">
   <property name="text">
    <string>load Picture</string>
   </property>
   <property name="toolTip">
    <string>load Picture</string>
   </property>
  </action>
  <action name="actionload_Folders">
   <property name="text">
    <string>load Folders</string>
   </property>
  </action>
  <action name="actionsingle_Folder">
   <property name="text">
    <string>single Folder</string>
   </property>
  </action>
  <action name="actionrekursive_Folders">
   <property name="text">
    <string>rekursive Folders</string>
   </property>
  </action>
  <action name="actionclear_Album">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>clear Album</string>
   </property>
  </action>
  <action name="showDefaultExifAct">
   <property name="text">
    <string>show default Exif meta data</string>
   </property>
  </action>
  <action name="clearDefaultExifAct">
   <property name="text">
    <string>clear default Exif meta data</string>
   </property>
  </action>
  <action name="showDefaultIptcAct">
   <property name="text">
    <string>show default IPTC meta data</string>
   </property>
  </action>
  <action name="clearDefaultIptcAct">
   <property name="text">
    <string>clear default Exif meta data</string>
   </property>
  </action>
  <action name="selectAllImagesAct">
   <property name="text">
    <string>select all Pictures</string>
   </property>
  </action>
  <action name="writeDefaultMetaToSelectedImagesAct">
   <property name="text">
    <string>write default Meta to selected Pictures</string>
   </property>
  </action>
  <action name="removeImagesAct">
   <property name="text">
    <string>remove selected Pictures from Album</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="hw_infoAct">
   <property name="text">
    <string>HW info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
